package.json

```diff
@@ -30,6 +30,7 @@
   },
   "homepage": "",
   "devDependencies": {
+    "@babel/plugin-proposal-class-properties": "^7.8.3",
```

Comment:
what's this?

Reply:
its from when i was playing around with dependencies; will remove